src/notifications/src/NotificationConsumer.test.ts

```diff
@@ -0,0 +1,23 @@
+import * as sendEmail from './index';
+import handleNotif from './NotificationConsumer';
+import * as cassandra from 'cassandra-driver';
+
+
+const message = {
+    detections: ['Joe'],
+    camera_id: 1,
+    frame_id: 2
+}
+

```

Comment:
I think you should make message a type:
https://www.typescriptlang.org/docs/handbook/declaration-files/by-example.html

Example that I have in this repo:
https://github.com/meowl-surveillance-system/meowl/blob/master/src/camera/mobile/nodemediaclient.type.ts

Reply:
I changed it so the message will be closer to what the exact kafka message sent looks like to better simulate the behavior so I made the type kafka.Message.